src/css/App.css

```diff
@@ -548,11 +549,16 @@ a:hover {
 }
 
 .thumbnail-generate-button {
-    margin-left: 400px;
```

Comment:
Just realized that the thumbnail generation button is positioned on the right. I like it!